### DISCUSSION TITLE
get_encrypted_app_ticket() function needs userdata, what is it?

### DISCUSSION THREAD

**Discussion by AivinXJ** [ValvePython/steam#407] [Q&A] [answered]

**Comment by AshladBP** [comment]
UP ! I have the same problem

**Comment by rossengeorgiev** [comment, marked as answer]
`userdata` is data the game wants encrypted in the ticket, and which can be verified by server, by decrypting the ticket.

https://partner.steamgames.com/doc/features/auth#encryptedapptickets
